.github/ISSUE_TEMPLATE/bugreport.yml

```diff
@@ -0,0 +1,102 @@
+name: Bug Report
+description: Report a bug building or running ungoogled-chromium
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before submitting this issue, please confirm if you should submit it to a [platform-specific issue tracker instead](//github.com/Eloston/ungoogled-chromium/blob/master/SUPPORT.md#platform-specific-support)
+
+        Otherwise, your issue may not be noticed. In addition, please read through the [SUPPORT.md](//github.com/Eloston/ungoogled-chromium/blob/master/SUPPORT.md) first
+  - type: dropdown
+    id: os
+    attributes:
+      label: OS/Platform
```

Comment:
"OS/Platform" should just be replaced with "Platform" pretty much everywhere.

Reply:
Why?

Reply:
> Why?

It's shorter, and "OS/Platform" is sorta redundant. Just my opinion though, it's up to you.

Reply:
One could be using Ubuntu (which is an OS, not a platform) and have ungoogled-chromium there. Also one could install Flatpak (which is more like a platform, than an OS) in Ubuntu and then use ungoogled-chromium from within Flatpak. Therefore I put "OS/Platform" there.